github/actions/multi_client.go

```diff
@@ -105,16 +96,6 @@ func (m *multiClient) GetClientFor(ctx context.Context, githubConfigURL string,
 
 	m.logger.Info("creating new client", "githubConfigURL", githubConfigURL, "namespace", namespace)
 
-	client, err := NewClient(
-		githubConfigURL,
-		&creds,
-		WithUserAgent(m.userAgent),
-		WithLogger(m.logger),
-	)
-	if err != nil {
-		return nil, err
-	}
-
 	m.clients[key] = &actionsClientWrapper{
```

Comment:
Maybe we can include in this PR not to have client wrapper
wrapper has a mutex that should guard refresh, but itt is never accessed since the client itself is guarding against refresh right? We don't really need a wrapper as far as I can tell

Reply:
LOVE IT ❤️